### ISSUE TITLE
2022 한국문화재재단 문화유산채널 뉴스레터 '금주의 문화유산' 운영

### ISSUE THREAD

**Issue by mia-slowalk** [slowalk/K-HERITAGE.TV.newsletter#1]

**Comment by mia-slowalk** [comment]
**[팀원 롤 분배]**

- **PM 미아** 
  - 뉴스레터 콘텐츠 기획&작성
  - 재단 커뮤니케이션
  - 제작 및 운영 스케줄 관리
  - 메일 발송
  - html 전달


- **디자이너 미솜**
  - 뉴스레터 템플릿 디자인
  - 섹션별 영상 썸네일 뉴스레터용으로 제작(템플릿화 할 예정)


- **내부 검수 초록&펭도**
  - 콘텐츠 검수

**Comment by mia-slowalk** [comment]
@pengdo-slowalk 펭도님 과업지시서 내용 검토 부탁드립니다!

[2022년 한국문화재재단 문화유산채널 뉴스레터 과업지시서.pdf](https://github.com/slowalk/K-HERITAGE.TV.newsletter/files/7861310/2022.pdf)

**Comment by pengdo-slowalk** [comment]
p.4 수정필요 
이메일 발송 대행(주4회) --> 월4회

그리고 p.4 뉴스레터 디자인 부분에 '4~5개의 영상별 카테고리'라고 카테고리 수가 명시되어 있는데 재단과 소통하신 부분이 맞는지 확인해주세요.

**Comment by mia-slowalk** [comment]
네! 혹시 수정은 제가 한 후에 다시 담당자님께 수정 내용과 과업지시서 파일을 전해드리면 되는걸까요? 혹은 수정 필요한 내용을 말씀드리기만 하면 되나요?

**Comment by pengdo-slowalk** [comment]
수정 내용이 많지 않으면 그 부분만 말씀드려주세요.

**Comment by mia-slowalk** [comment]
- 제출 필요 서류
  - 착수계
  - 용역예정공정표
  - 용역책임자와 참여 기술자의 이력서 및 명단(전달드린 용역참여자 명단에 용역책임자와 참여 기술자의 이력서 모두 전달)
  - 정보누츨금지 확약서 및 보안각서(총괄책임자 및 실무담당자 전원 작성) 

- 추가 제출 서류
  - 착수계제출 공문
  - 산출내역서
  - 사업수행계획서(전달드린 과업지시서 기반으로 작성)

- 우선 파일로 전달하고 확인 완료 되면 직인을 날인한 원본으로 전달

**Comment by mia-slowalk** [comment]
- [한국문화재재단 주간 뉴스레터 제안서](https://docs.google.com/presentation/d/1hSejFu40jwm0W8lRZAeOUnqtkNWG_B-YPsJaXToWIn0/edit?usp=sharing)
  - 작성 완료했습니다! 문서 내에 댓글로 피드백 부탁드려요:)

**Comment by pengdo-slowalk** [comment]
시간대별로 본문을 구성한 것, 월말 결산 제안한 것 탁월하네요! 사소한 수정사항은 이따 말씀드릴게요. 고생많으셨어요!

**Comment by mia-slowalk** [comment]
**[스티비 결제]**

- 연간 결제 안내 사항
> 5️⃣ 연간 결제 사용 중 높은 요금제로 변경하려면(라이트 요금제 → 스탠다드 요금제) 연간 결제 신청 양식을 통해 요금제 변경을 신청해야 합니다. 요금제 변경 시점부터 계약 종료일까지의 남은 이용기간에 대한 추가요금이 발생합니다.
> [추가요금 계산 방법 보기](http://help.stibee.com/ko/articles/1040850-%EC%8A%A4%ED%83%A0%EB%8B%A4%EB%93%9C-%EC%9A%94%EA%B8%88%EC%A0%9C-%EC%82%AC%EC%9A%A9-%EC%A4%91-%EC%9A%94%EA%B8%88%EA%B5%AC%EA%B0%84%EC%9D%B4-%EB%86%92%EC%95%84%EC%A1%8C%EC%9D%84-%EB%95%8C-%EC%B6%94%EA%B0%80%EC%9A%94%EA%B8%88%EC%9D%80-%EC%96%B4%EB%96%BB%EA%B2%8C-%EA%B3%84%EC%82%B0%EB%90%98%EB%82%98%EC%9A%94#--)
> 
> 6️⃣ 연간 결제 사용 중 낮은 요금제로 변경하는 것(스탠다드 2,501-5,000 → 스탠다드 1,001-2,500 구간)은 불가능합니다.
> 
> 7️⃣ 스탠다드 요금제 사용 중 구독자 수가 늘어난 경우, 연간 결제 신청 양식을 통해 요금구간 변경을 신청해야 합니다. 요금구간 변경 시점부터 계약 종료일까지의 남은 이용기간에 대한 추가요금이 발생합니다. 요금구간을 높이는 경우 원하시는 날짜를 기준으로 추가요금 견적서를 보내드립니다.
> [추가요금 계산 방법 보기](http://help.stibee.com/ko/articles/1040850-%EC%8A%A4%ED%83%A0%EB%8B%A4%EB%93%9C-%EC%9A%94%EA%B8%88%EC%A0%9C-%EC%82%AC%EC%9A%A9-%EC%A4%91-%EC%9A%94%EA%B8%88%EA%B5%AC%EA%B0%84%EC%9D%B4-%EB%86%92%EC%95%84%EC%A1%8C%EC%9D%84-%EB%95%8C-%EC%B6%94%EA%B0%80%EC%9A%94%EA%B8%88%EC%9D%80-%EC%96%B4%EB%96%BB%EA%B2%8C-%EA%B3%84%EC%82%B0%EB%90%98%EB%82%98%EC%9A%94#--)
> 
> 8️⃣ 스탠다드 요금제 사용 중 구독자 수가 줄어들어도 요금구간이 변경되지 않습니다. 따라서 요금구간 변경에 대한 환불이나 보상은 하지 않습니다.

- 결제 내용
  - 현재 구독자 312명
  - 스탠다드 요금제(0-500명): 연간 48,000원

**Comment by mia-slowalk** [comment]
- 프로젝트 관련 파일 아카이빙은 [여기](https://drive.google.com/drive/folders/17AafTyoDSn83tX5Mc2-8x6DcksvCiNbc?usp=sharing)

**Comment by mia-slowalk** [comment]
**[개인정보 보호 교육 실태]**
- 관련 제출 파일은 [여기](https://drive.google.com/drive/folders/193faLK1oDjrQIGJ6kGIkdxwmFV_nVn5q?usp=sharing)에서 다운로드(미아 작성)
- 수탁업체 대상 교육결과 보고(미디어콘테츠팀)
  - 교육사진(2~3장) 등 문서 서식에 맞춰서 작성
  - 총 1회 전달

- 월간 개인정보 처리실태 점검표
  - 담당자 날인하여 스캔본 매월 전달, 사업 종료 전 전체 날인 본 우편으로 전달
  - 스캔본 전달 시, 특이사항 및 보안점검 방법(ex, 보안체크리스트를 활용하여 개인정보 관리, 000프로그램을 이용하여 보안 관리 감독 등) 등 전달
  
- 차주 목요일(3월 3일)까지 전달

**Comment by mia-slowalk** [comment]
**[참고]** 뉴스레터 운영담당자가 바뀐다고 해요. 그래서 정해질 때까지 김한태 팀장님과 소통 진행하기로 되었어요. (유우경 PM님의 퇴사인지, 정확한 사유는 모르겠네요..)
